packages/fast-components-react-base/src/tabs/tabs.props.ts

```diff
@@ -5,6 +5,22 @@ import {
 } from "@microsoft/fast-components-class-name-contracts-base";
 import { Orientation } from "@microsoft/fast-web-utilities";
 
+export interface TabConfig {
```

Comment:
What is this? It doesn't seem like it is actually used.

Reply:
Not sure honestly. This branch has existed for awhile but I don't remember adding that. At any rate its been removed.